docs/src/usage/filter.md

```diff
@@ -0,0 +1,58 @@
+# Filtering
+
+Filtering in `Neuroimaging.jl` is performed using the `DSP.jl` backend.
+
+There are two main ways to use the filter function:
+1) Modality specific filter
+2) Custom defined filter
+
+Let's load some data and apply the filter
+
+```@example filter
+using DisplayAs # hide
+using Neuroimaging, DataDeps, StatsBase
+
+data_path = joinpath(
+    datadep"ExampleSSR",
+    "Neuroimaging.jl-example-data-master",
+    "neuroimaingSSR.bdf",
+)
+
+s = read_EEG(data_path)
+s.data = s.data .- StatsBase.mean(s.data,dims=1) # remove DC offset for better plotting
+```
+
+
+## 1) Modality specific filter
+We offer high and lowpass filter with reasonable defaults for each modality (e.g. `GeneralEEG` a FIR filter,`SSR` a filtfilt Butterworth)
+
+```@example filter
+s_hp = filter_highpass(s,2)
+s_lp = filter_lowpass(s,5) # extreme value to show LP effect in plot
+
+plot(s.data[:,1],label="raw")
+plot!(s_hp.data[:,1],label="highpass")
+plot!(s_lp.data[:,1],label="lowpass")
```

Comment:
Don't access fields directly. if you want access to the data field then write a function called `data(s)` (on my todo list anyway). See sampling rate example below.

Can you use the existing plotting functions? E.g. https://rob-luke.github.io/Neuroimaging.jl/dev/assr/examples/#Visualise-processed-continuous-data They are pretty outdated, we may need to update them. They definitely need to be renamed to something standard like plot and plot! If they dont meet you needs, can you raise an issue with dot points as to what needs to be added?

Reply:
I try to be helpful. See #169